components/Premium/PremiumFeatures/PremiumFeatures.tsx

```diff
@@ -22,62 +29,143 @@ function PremiumFeatures() {
-                        <td>{xIcon}</td>
-                        <td>{checkIcon}</td>
+                        <td className={styles.featureColumn}>Top 3 Kat flips</td>
+                        {xIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                    </tr>
+                    <tr>
+                        <td className={styles.featureColumn}>Top 3 Craft flips</td>
+                        {xIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
                     </tr>
                     <tr>
                         <td className={styles.featureColumn}>Full access to flipper filters</td>
-                        <td>{xIcon}</td>
-                        <td>{checkIcon}</td>
+                        {xIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                    </tr>
+                    <tr>
+                        <td className={styles.featureColumn}>
+                            List of low supply items (<Link href="/lowSupply">here</Link>)
+                        </td>
+                        {xIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
                     </tr>
                     <tr>
-                        <td className={styles.featureColumn}>Up to 100 notifications</td>
-                        <td>{xIcon}</td>
-                        <td>{checkIcon}</td>
+                        <td className={styles.featureColumn}>Support the project</td>
+                        {xIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
                     </tr>
                     <tr>
                         <td className={styles.featureColumn}>Priority feature request</td>
-                        <td>{xIcon}</td>
-                        <td>{checkIcon}</td>
+                        {xIconElement}
+                        {xIconElement}
+                        {checkIconElement}
+                        {checkIconElement}
                     </tr>
                     <tr>
-                        <td className={styles.featureColumn}>
-                            List of low supply items (<Link href="/lowSupply">here</Link>)
+                        <td className={styles.featureColumn}>Discord role</td>
+                        <td className={styles.premiumProductColumn}>-</td>
+                        <td className={styles.premiumProductColumn}>-</td>
+                        <td className={styles.premiumProductColumn}>Flipper</td>
+                        <td className={styles.premiumProductColumn}>Sniper</td>
```

Comment:
Add a note stating that the role will be given out in the second week of September, the bot currently can't do it

Reply:
or just make it say flipper for now